### PR TITLE
Add SizzurpMods

### PR DESCRIPTION
**Pull request by LilSizzurp**

### CODE REVIEW

**Comment by VukAnd** [comment]
Also, you forgot to actually add a link to the mod in the README

**Comment by LilSizzurp** [comment]
https://github.com/DesktopGooseUnofficial/SizzurpMods

File: mods/SizzurpMods.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# SizzurpMods
+# 🙂 SizzurpMods
```
Comment (VukAnd) [inline mods/SizzurpMods.md]: Cool, could you just add the rating to the README.md as well?

Comment (LilSizzurp) [reply mods/SizzurpMods.md]: are u blind

**Pull request merged**